packages/ceramic-core/src/__tests__/__validation__/__schemas__/crypto-account-links.json

```diff
@@ -1,7 +1,7 @@
 {
   "$schema": "http://json-schema.org/draft-07/schema#",
   "type": "object",
-  "title": "CryptoAccountLinks",
+  "title": "CryptoAccountCaip10Links",
```

Comment:
Wasn't totally sure if this change made sense, lmk if I should put it back the way it was

Reply:
Not sure if these schemas are still needed here. Are they used for testing maybe @simonovic86 ?
Anyhow, I'd just name this `CryptoAccounts`.

Reply:
@oed yep, they are used for testing only 👍